composition-js/src/__tests__/connectors.test.ts

```diff
@@ -26,77 +26,75 @@ describe("connect spec and join__directive", () => {
       }
   `;
 
-  const expectedSupergraphSdl = `
-    "schema
-      @link(url: \\"https://specs.apollo.dev/link/v1.0\\")
-      @link(url: \\"https://specs.apollo.dev/join/v0.5\\", for: EXECUTION)
-      @link(url: \\"https://specs.apollo.dev/connect/v0.3\\", for: EXECUTION)
-      @join__directive(graphs: [WITH_CONNECTORS], name: \\"link\\", args: {url: \\"https://specs.apollo.dev/connect/v0.1\\", import: [\\"@connect\\", \\"@source\\"]})
-      @join__directive(graphs: [WITH_CONNECTORS], name: \\"source\\", args: {name: \\"v1\\", http: {baseURL: \\"http://v1\\"}})
-    {
-      query: Query
-    }
+  const expectedSupergraphSdl = `schema
+  @link(url: "https://specs.apollo.dev/link/v1.0")
+  @link(url: "https://specs.apollo.dev/join/v0.5", for: EXECUTION)
+  @link(url: "https://specs.apollo.dev/connect/v0.4", for: EXECUTION)
```

Comment:
These changes should all be reindentation, except for this connect/v0.3 => connect/v0.4 change.